community/calcurse/build

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+sed -i test/Makefile.in -e '/ical-009.sh \\/d'
```

Comment:
We typically prefer this syntax: `sed -i '/ical-009.sh \\/d' test/Makefile.in`

Reply:
Noted.